content/information-distribution/article.md

```diff
@@ -12,22 +12,15 @@ signature coming from its source, and be published on the publication name space
 that has been declared by its source. The article **can** also include optional
 properties, such as the item's description, read duration, NSFW marker, etc..
 
-The signature **must** be generated from one of the source's public signature
-verification keys (using the source's declared signing algorithm), and a string
-containing all of the item's information, and **must** be formatted the same way
-as follows:
+If the source has one or more signature keys, the article JSON object **must**
+be signed with one of the source's keys, using the [Canonical JSON
```

Comment:
Might be worded differently, my take on it being `using the method [described in the relevant part of this documentation].` or something similar. I'm not a huge fan of naming it "the Canonical JSON method".